frontend/js/src/explore/fresh-releases/components/ReleaseFilters.tsx

```diff
@@ -221,6 +233,42 @@ export default function ReleaseFilters(props: ReleaseFiltersProps) {
           Check out all releases worldwide, or just from artists you&apos;ve
           listened to before, with &apos;for you&apos;.
         </p>
+        <FontAwesomeIcon
+          icon={faSquareRss}
+          size="lg"
+          className="feed-button"
+          data-toggle="modal"
+          data-target="#SyndicationFeedModal"
+          style={{ color: "#fff" }}
+          onClick={() => {
+            if (pageType === PAGE_TYPE_SITEWIDE) {
+              NiceModal.show(SyndicationFeedModal, {
+                feedTitle: `Site-wide Fresh Releases`,
+                options: [
+                  {
+                    label: "Days",
+                    key: "days",
+                    type: "number",
+                    values: [
+                      { id: "3", value: "3" },
+                      { id: "7", value: "7" },
+                      { id: "14", value: "14" },
+                      { id: "30", value: "30" },
+                    ],
+                    defaultIndex: 0, // Default to 3 days
+                  },
+                ],
+                baseUrl: `https://listenbrainz.org/syndication-feed/fresh-releases`,
+              });
+            } else if (pageType === PAGE_TYPE_USER) {
+              NiceModal.show(SyndicationFeedModal, {
+                feedTitle: `User-specific Fresh Releases`,
+                options: [],
+                baseUrl: `https://listenbrainz.org/syndication-feed/user/${currentUser.name}/fresh-releases`,
```

Comment:
I'm worried here that `currentUser` might be undefined.
In some other cases you used `currentUser?.name` with the question mark making it safe.
This part of the page shouldn't be accessible by a user that is not logged in, but better make sure it is accessed safely.